components/analytics/compact-dashboard.tsx

```
@@ -0,0 +1,1298 @@
+                                                                                    }}
+                                                                                />
+                                                                            </Pie>
+                                                                            <Tooltip
+                                                                                formatter={(value, name) => {
+                                                                                    const numValue = Number(value);
+                                                                                    const percentage = platformTotal > 0 ?
+                                                                                        (numValue / platformTotal) * 100 : 0;
+
+                                                                                    return [
+                                                                                        <div className="flex flex-col text-white">
+                                                                                            <span className="font-medium text-white">{formatNumber(numValue)} bots</span>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)